data/themes/darktable.css

```diff
@@ -143,6 +143,10 @@
 @define-color thumbnail_hover_bg_color @grey_85;
 @define-color thumbnail_hover_fg_color @grey_90;
 @define-color filmstrip_bg_color @darkroom_bg_color;
+@define-color timeline_bg_color @grey50;
```

Comment:
an underscore is missing. Annyway,  I think now it would be better to not have specific background for timeline and just use filmstrip (while allowing the possibly to change it. So, I suggest this line instead:

`@define-color timeline_bg_color @filmstrip_bg_color;`

Reply:
well, I agree for the underscores.... stupid mistake !

but `timeline_bg_color`  and `filmstrip_bg_color` should stay different : the filmstrip background is used for the "real background of the timeline (= the part where there's nothing to draw or between the year/month/... rectangle)
the timeline background is used to draw the year/month/... rectangles

if we want to keep the same colors as now codewise, we should use : 
- `timeline_bg_color` => `thumbnail_bg_color`
- `timeline_fg_color` => `thumbnail_hover_bg_color`

but as this "reused" colors doesn't really means anything, I've preferred to use the 'colors' directly...

What would be the best for you to define those timeline_... colors : direct @grey_... or @thumbnail_... ?? (I don't really care, it's up to you :)

Reply:
I know what they mean and I really think there's really not need to set a different background for timeline by default as timeline is well defined and visible by itself. So different background color to set limits is not needed I think. That's why i suggest that. So I suggest to chance timeline color scheme.

About using @filmstrip_bg_color instead of @grey_... is because I choose to use same color as filmstrip here. So changing first (if needed in the future or something else) will change second. When we choose to set same color for 2 things, I think it's the best. Otherwise, use of course @grey_... setting.

Reply:
ok, I've not understood that you wanted to change the colorscheme... no problem for me, except maybe that with the 2 background colors equals, we miss the separations... but that's not a blocking point either, and I trust you blindly about colors and co ;)

for the record, here's a before/after screenshots : 
![timeline_before](https://user-images.githubusercontent.com/1818223/130369556-ab6f669b-fe1b-47cb-aed5-3e7ea34c533b.png)
![timeline_after](https://user-images.githubusercontent.com/1818223/130369560-3dfd9aa1-aecc-4346-9be9-e2638010efb7.png)

now I need to address your second point about direct change and I'll update the PR

Reply:
> no problem for me, except maybe that with the 2 background colors equals, we miss the separations... but that's not a blocking point either, and I trust you blindly about colors and co ;)

I never use timeline so maybe you're right to let 2 different background colors. On what I've test/check, same background seems good but maybe something safer with 2 different background is better. Anyway, actually difference is too strong, so I would finally suggest to use a background color near filmstrip to let a little difference. I suggest that line so instead:

`@define-color timeline_bg_color shade(@filmstrip_bg_color, 0.9);`

And with suggested @text_color setting for foreground color. I hope it's clear and good for you.